### PR TITLE
Moved GWMS install to a single script slimmed the testbed image

### PR DESCRIPTION
**Pull request by mambelli**
Moved GWMS install to a single script and added a developer install script to keep the testbed leaner and better test RPM dependencies

### CODE REVIEW

File: workspaces/testbed-workspace/scripts/install-developer.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/bash
+
+# SPDX-FileCopyrightText: 2020 Fermi Research Alliance, LLC
+# SPDX-License-Identifier: Apache-2.0
+
+# Install development software
+# Used to minimize the software installed in the base image (testbed-workspace)
+
+# Development tools
+dnf install -y python3 wget git jq vim zsh sudo psmisc bind-utils mlocate
```
Comment (BrunoCoimbra) [inline workspaces/testbed-workspace/scripts/install-developer.sh:10]: `wget` is already installed on the Dockerfile. Do we need it here?

**Review by BrunoCoimbra — APPROVED** [review]
Looks good to go. I made a couple of comments, but no show stoppers.

File: workspaces/compose-testbed.yml [hunk 2]
```diff
@@ -17,6 +18,8 @@ services:
         - ${IMAGE_NAMESPACE-glideinwms}/ce-workspace:${IMAGE_LABEL-latest}
       dockerfile: ce-workspace/Dockerfile
     image: ${IMAGE_NAMESPACE-glideinwms}/ce-workspace:${IMAGE_LABEL-latest}
+    volumes:
+      - ${GWMS_PATH-gwms-dev-local}:/opt/gwms
```
[the remark this answers is not in the document]
Comment (mambelli) [reply workspaces/compose-testbed.yml:22]: To share files. In my tests I had a file on the Factory to test logging that I did not transfer in the glidein (file section) and wanted to use to test the log system. I ended up doing copy/paste in terminal but the shared dir would have been convenient.

File: workspaces/compose.yml [hunk 3]
```diff
@@ -15,6 +15,8 @@ services:
         - ${IMAGE_NAMESPACE-glideinwms}/ce-workspace:${IMAGE_LABEL-latest}
       dockerfile: ce-workspace/Dockerfile
     image: ${IMAGE_NAMESPACE-glideinwms}/ce-workspace:${IMAGE_LABEL-latest}
+    volumes:
+      - ${GWMS_PATH-gwms-dev-local}:/opt/gwms
```
[the remark this answers is not in the document]
Comment (mambelli) [reply workspaces/compose.yml:19]: see other

File: workspaces/testbed-workspace/scripts/install-developer.sh (hunk 1 above)
Comment (mambelli) [reply workspaces/testbed-workspace/scripts/install-developer.sh:10]: no, good catch. Earlier was not in the docker file. I'll merge the PR (to keep the approval) and remove wget

**Pull request merged**